gradle/libs.versions.toml

```diff
@@ -97,4 +97,13 @@ hamcrest = { group = "org.hamcrest", name = "hamcrest", version = "3.0" }
 robolectric = { group = "org.robolectric", name = "robolectric", version = "4.13" }
 uiautomator = { group = "androidx.test.uiautomator", name = "uiautomator", version = "2.3.0" }
 
-[plugins]
\ No newline at end of file
+[plugins]
+androidApplication = { id = "com.android.application" }
```

Comment:
Is there a way to declare the plugin versions here so we don't still need these declared in the root `build.gradle`?

Reply:
Yes, I even attempted to do that, but since it was quite complex (requiring a lot of additional changes) and some of the functions I would need to use are still marked as unstable, I decided to hold off for now.